README.md

```diff
@@ -67,6 +67,12 @@ nix-shell
 
 ### Building and running
 
+First, initialize the repository:
+
+```sh
+git clone https://github.com/au-ts/libvmm
+cd libvmm
+git submodule update --init
```

Comment:
I messed up the suggestion sorry, there should still be three back ticks at the end and then a new line.

Reply:
Got it, should be good now.